src/lib/util/Database/ConfigurableKey.ts

```diff
@@ -0,0 +1,157 @@
+import type { CustomGet } from '@lib/types/Shared';
```

Comment:
This file is mostly duplicated in `src/lib/database/settings/ConfigurableKey.ts`